addons/l10n_my_edi/__manifest__.py

```diff
@@ -0,0 +1,30 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+{
+    'name': 'Malaysia - E-invoicing',
+    'countries': ['my'],
+    'version': '1.0',
+    'category': 'Accounting/Localizations/EDI',
+    'icon': '/account/static/description/l10n.png',
+    "summary": "E-invoicing using MyInvois",
+    'description': """
+    This modules allows the user to send their invoices to the MyInvois system.
+    """,
+    # The export does not depend on the pint format, but we need to reuse the fields defined there.
+    'depends': ['l10n_my', 'l10n_my_ubl_pint', 'account_edi_proxy_client'],
+    'data': [
+        'data/ir_cron.xml',
+        'data/l10n_my_edi.industry_classification.csv',
+        'data/my_ubl_templates.xml',
+        'security/ir.model.access.csv',
+        'views/account_move_view.xml',
+        'views/account_tax_view.xml',
+        'views/product_template_view.xml',
+        'views/res_company_view.xml',
+        'views/res_config_settings_view.xml',
+        'views/res_partner_view.xml',
+        'wizard/account_move_send_views.xml',
+        'wizard/l10n_my_edi_status_update_wizard.xml',
+    ],
+    'installable': True,
```

Comment:
Should we make it auto-install? (according to JCO, our dear enlightened CEO has requsted in the past for EDIs to be auto-installed)
Note that if we do that, we should check that this doesn't break anything in the 16.0 -> 17.0 migration. But as far as I see, it shouldn't break anything.

Reply:
I usually prefer not to, but if you think it is better I can.
We just released an edi in Vietnam and I wasn't asked to make it auto install :thinking:

Reply:
For me, it's fine to leave it as it is now :+1: